### PR TITLE
version files now expose a call that can be called by apps that don't do index import

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

This change adds a function that can be called manually by the consumers who are not using the 'office-ui-fabric-react' at the root. Those consumers will call this function manually somewhere to record the version of fabric that is bundled for them.

### CODE REVIEW

File: scripts/generate-version-files.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ packageJsons.forEach(packageJsonPath => {
       `// ${packageJson.name}@${packageJson.version}
 // Do not modify this file, the file is generated as part of publish. The checked in version is a placeholder only.
 import { setVersion } from '@uifabric/set-version';
-setVersion('${packageJson.name}', '${packageJson.version}');`
+export const version = () => setVersion('${packageJson.name}', '${packageJson.version}');
```
Comment (dzearing) [inline scripts/generate-version-files.js]: Suggestions; 
1. This should export a `setVersion` method. Calling it `version` is a little obscure. (rename the file too)
2. It shouldn't be called in this file.
3. Import setVersion and call it in the index file.

For partners that use `/lib/*` imports, they can just do this:
```tsx
import { setVersion } from 'oufr/lib/setVersion';
setVersion();
```
Then there are no side effects for them to consider.

File: common/changes/@uifabric/set-version/version_2018-09-27-16-50.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "changes": [
+    {
+      "packageName": "@uifabric/set-version",
+      "comment": "version files now expose a call that can be called by apps that don't do index import",
+      "type": "patch"
+    }
+  ],
+  "packageName": "@uifabric/set-version",
+  "email": "kchau@microsoft.com"
+}
```
Comment (cliffkoh) [inline common/changes/@uifabric/set-version/version_2018-09-27-16-50.json]: Should we update the baseline so it more accurately reflects this not-insignificant change?

**Comment by cliffkoh** [comment]
Hmm after thinking a bit more, I strongly recommend updating the baselines. It's probably worth it too to force this change to be published across all the packages as well.

File: common/changes/@uifabric/set-version/version_2018-09-27-16-50.json (hunk 2 above)
Comment (kenotron) [reply common/changes/@uifabric/set-version/version_2018-09-27-16-50.json]: Okay I've reverted that generator change so this isn't necessary anymore

**Pull request merged**